interactions.go

```diff
@@ -326,6 +326,18 @@ type ApplicationCommandInteractionData struct {
 	TargetID string `json:"target_id"`
 }
 
+// Get the interaction data option by name
+func (d ApplicationCommandInteractionData) GetOption(name string) (option *ApplicationCommandInteractionDataOption) {
```

Comment:
Could we have these return an exists boolean as well?

Reply:
Well you can just null-check it, an exists boolean isn't needed i suppose.